### ISSUE TITLE
Link to send account details or new password to the user

### ISSUE THREAD

**Issue by mike-podolskiy90** [gbif/ipt#2091]
Add a mailto links to the notification messages:

- [ ] When new user created
- [ ] When user account's password reset

Example:
![Screenshot 2023-09-06 at 13 31 02](https://github.com/gbif/ipt/assets/24795387/60e7ac6a-afe2-440c-919a-48d244cd33b7)

**Comment by EstebanMH-SiB** [comment]
Is there a way to translate the default message for the password? I do not see it in crowdin and we would like to send this messages in spanish

**Comment by mike-podolskiy90** [comment]
You're right, it's not added to the translatable variables. I'll add it and let you know. New IPT version should be released to address that though.

**Comment by EstebanMH-SiB** [comment]
@mike-podolskiy90 Any update in this one? We want to start using this functionality

**Comment by mike-podolskiy90** [comment]
@EstebanMH-SiB Thank you for pinging me. I'm afraid it wasn't implemented - it got lost among other issues. I've logged a task for it https://github.com/gbif/ipt/issues/2787
